ui/webui/src/components/installation/InstallationProgress.jsx

```diff
@@ -59,6 +58,25 @@ export class InstallationProgress extends React.Component {
                                             this.props.onAddErrorNotification
                                         );
                             }
+                            console.log("MESSAGE");
+                            console.log(step);
+                            console.log(message);
+                            // storage
+                            if (step <= 3) {
```

Comment:
Will these magical constants be addressed somehow in the future? If yes, there should be a FIXME comment.

Reply:
Oh, right, that's a really good point and really should have been noted in the commit message or a code comment. 

So basically the way how the progress stepping is implemented right now is rather tightly coupled to the MVP use case, primarily to get feedback for using the ProgressStepper in the Web UI. If the feedback is positive and we keep it, this will definitely have a proper API. 

Likely both for getting a list of phases (as those can very well change based on whats being installed) as well as proper signals when the installation process switches to the next phase.